lib/App.server.js

```diff
@@ -29,6 +29,7 @@ App.prototype._init = function(options) {
   this.scriptMapFilename = null;
   this.scriptBaseUrl = (options && options.scriptBaseUrl) || '';
   this.scriptMapBaseUrl = (options && options.scriptMapBaseUrl) || '';
+  this.scriptCrossOrigin = (options && options.scriptCrossOrigin) || '';
```

Comment:
Changed the default to false. As we discussed over chat, leaving the option name as `scriptCrossOrigin`, as it's essentially controlling `scriptElement.crossOrigin`.